src/ui.rs

```diff
@@ -1102,16 +1356,26 @@ impl App {
                 InputMode::Normal => Style::default(),
                 InputMode::Editing => Style::default().fg(Color::Yellow),
                 InputMode::CreatingStory { .. } => Style::default().fg(Color::Green),
+                InputMode::QuickReply { .. } => Style::default().fg(Color::Cyan),
+                InputMode::MessageComposition { .. } => Style::default().fg(Color::Magenta),
             };
 
             let input_text = match &self.input_mode {
                 InputMode::Normal => {
                     match &self.view_mode {
-                        ViewMode::Channels => "Press 'i' to enter input mode, Enter to view channel stories, ←/→ to navigate, ↑/↓ to scroll, Ctrl+S to toggle auto-scroll, 'c' to clear output, 'q' to quit".to_string(),
-                        ViewMode::Stories(_) => "Press 'i' to enter input mode, Enter to view story, Esc to return to channels, ←/→ to navigate, ↑/↓ to scroll, Ctrl+S to toggle auto-scroll, 'c' to clear output, 'q' to quit".to_string(),
+                        ViewMode::Channels => "Press 'i' to enter input mode, 'r' for quick reply, 'm' for message compose, Enter to view channel stories, ←/→ to navigate, ↑/↓ to scroll, Ctrl+S to toggle auto-scroll, 'c' to clear output, 'q' to quit".to_string(),
```

Comment:
These messages to put in the footer pane are now too long and do not fit.  Image attached,.

<img width="1286" height="40" alt="Image" src="https://github.com/user-attachments/assets/bb308db9-a89d-42f9-a228-543ecd62e72e" />